perceval/simulators/simulator.py

```diff
@@ -392,6 +397,8 @@ def _probs_svd_fast(self, input_dist, p_threshold, progress_callback: Callable |
                 exec_request = progress_callback(progress, 'recombine distributions')
                 if exec_request is not None and 'cancel_requested' in exec_request and exec_request['cancel_requested']:
                     raise RuntimeError("Cancel requested")
+        if self._logical_perf > 0:
```

Comment:
I think you need `self._physical_perf > 0` if you want to avoid division by 0 (anyway, given the definition of the logical perf, it doesn't make sense to define it if physical perf is 0)